src/components/skip-link/index.md.njk

```diff
@@ -15,7 +15,9 @@ Use the skip link component to help keyboard-only users skip to the main content
 
 ## When to use this component
 
-All GOV.UK pages must include a skip link in the header.
+All GOV.UK pages must include a skip link. Place the skip link immediately after the opening `<body>` tag.
+
+Do not wrap the skip link in a `<nav>` region, or move it inside the header. Some automated accessibility testing tools may warn that the skip link element is not inside a landmark. This warning does not apply to skip links, so you can ignore it.
```

Comment:
I think teams would only wrap or move the link _because of_ the warning, so would it make sense to swap this around? 🤔  It feels slightly weird to tell teams not to do a thing they probably wouldn't do before explaining why.

```suggestion
Some automated accessibility testing tools may warn that the skip link element is not inside a landmark. This warning does not apply to skip links, so you can ignore it. Do not wrap the skip link in a `<nav>` region, or move it inside the header.
```

Reply:
Makes sense! Thanks.